dimod/constrained.py

```diff
@@ -522,7 +566,18 @@ def from_discrete_quadratic_model(cls, dqm: DiscreteQuadraticModel, *,
     def from_quadratic_model(cls, qm: Union[QuadraticModel, BinaryQuadraticModel]
                              ) -> 'ConstrainedQuadraticModel':
         """Construct a constrained quadratic model from a quadratic model or
-        binary quadratic model."""
+        binary quadratic model.
+
```

Comment:
I think some text here explaining why it just sets the objective might be useful.

Reply:
Done. Also added a constraint to the example